feature/gribi/otg_tests/gribi_flush/README.md

```diff
@@ -0,0 +1,165 @@
+TE-18.4 - gRIBI Flush operation with Live traffic
+
+## Summary
+This test verifies when gRIBI flush operation is issued all the gRIBI rules are removed leaving the static routes intact.
+Ongoing traffic flows using gRIBI rules will switch to static route.
+
+## Topology
+
+* [`featureprofiles/topologies/atedut_2.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Test setup
+
+TODO: Complete test environment setup steps
+
+inner_ipv6_dst_A = "2001:aa:bb::1/128"
+inner_ipv6_dst_B = "2001:aa:bb::2/128"
+inner_ipv6_default = "::/0"
+
+ipv4_inner_dst_A = "10.5.1.1/32"
+ipv4_inner_dst_B = "10.5.1.2/32"
+ipv4_inner_default = "0.0.0.0/0"
+
+outer_ipv6_src =      "2001:f:a:1::0"
+outer_ipv6_dst_A =    "2001:f:c:e::1"
+outer_ipv6_dst_B =    "2001:f:c:e::2"
+outer_ipv6_dst_def =  "2001:1:1:1::0"
+outer_dst_udp_port =  "6635"
+outer_dscp =          "26"
+outer_ip-ttl =        "64"
+
+## Procedure
+
+### TE-18.4.1 Perform gRIBI flush operation to verify traffic switches to backup prefix match rule.
+
+#### gRIBI RPC content
+
+The gRIBI client should send this proto message to the DUT to create AFT
```

Comment:
is the intent to use the same programming as TE-18.1?  If yes, just reference it rather than cut-paste.